java/mvn/Dockerfile.8-jdk-alpine

```diff
@@ -0,0 +1,26 @@
+FROM openjdk:8-jdk-alpine
+
+RUN apk add --no-cache curl tar bash
```

Comment:
make this "RUN apk add --update --no-cache curl tar bash"

Always need either "apk update" or "apk add --update" once per dockerfile. This fixes my build error.